scipy/signal/_whittaker.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
+from scipy.linalg import solve, toeplitz
+
+
+# TODO:
+# 1) Add parameter `order` for the order of the difference penalty
+# 2) C code for _solve_WH_order2_fast
+# 3) Use sparse/banded arrays for _solve_WH_order_direct
+# 4) GCV for lamb
+# 5) Case weights
+
+def whittaker_handerson(signal, lamb = 0.0):
+    """
```

Comment:
when running the testsuite locally I saw:

```
scipy/signal/_whittaker.py:13: SyntaxWarning: invalid escape sequence '\l'
```

Reply:
I saw it, but what does it mean? My editor (VSCode) does not show any difference to anywhere else in the scipy source code. Should I just put the first line text directly behind `"""`?

Reply:
Got it, it was the LaTeX.